src/main/java/appeng/client/gui/AEBaseGui.java

```diff
@@ -83,17 +83,17 @@
 
 public abstract class AEBaseGui extends GuiContainer
 {
-	public static boolean switchingGuis;
-	protected final List<InternalSlotME> meSlots = new LinkedList<InternalSlotME>();
+	private static boolean switchingGuis;
+	private final List<InternalSlotME> meSlots = new LinkedList<InternalSlotME>();
 	// drag y
-	final Set<Slot> drag_click = new HashSet<Slot>();
-	final AppEngRenderItem aeRenderItem = new AppEngRenderItem();
-	protected GuiScrollbar myScrollBar = null;
-	boolean disableShiftClick = false;
-	Stopwatch dbl_clickTimer = Stopwatch.createStarted();
-	ItemStack dbl_whichItem;
-	Slot bl_clicked;
-	boolean useNEI = false;
+	private final Set<Slot> drag_click = new HashSet<Slot>();
+	private final AppEngRenderItem aeRenderItem = new AppEngRenderItem();
+	private GuiScrollbar myScrollBar = null;
+	private boolean disableShiftClick = false;
+	private Stopwatch dbl_clickTimer = Stopwatch.createStarted();
+	private ItemStack dbl_whichItem;
+	private Slot bl_clicked;
+	private final boolean useNEI = false;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "useNEI" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)